### PR TITLE
Siemens Industrial Edge

### PR DESCRIPTION
**Pull request by katarzyna-koltun-mx**

### CODE REVIEW

File: content/en/docs/developerportal/deploy/mendix-app-on-industrial-edge.md [hunk 1]
```diff
@@ -10,216 +10,34 @@ tags: ["Mendix app", "Industrial Edge", "edge", "docker image", "Mendix Buildpac
...
+Mendix on Edge empowers factory operators to create custom applications with the Mendix low-code platform that run locally as Edge Apps. Among its other tools, Industrial Edge provides the [Industrial Edge Plugin for Mendix Studio Pro](https://marketplace.mendix.com/link/component/218436), a tool extending Mendix Studio Pro seamlessly on a development PC which supports the developer with publishing Mendix apps into the Industrial Edge Environment.
 
-This documentation is meant for Mendix developers who want to use their apps in the Industrial Edge environment.
+This documentation is meant for Mendix developers who want to develop and use their apps in the Industrial Edge environment.
 
 ## 2 Prerequisites
 
-To convert your Mendix app to an Industrial Edge App, the following is required:
+For information about the dependencies and prerequisites to use Mendix on Edge, see [Mendix on Edge](https://docs.eu1.edge.siemens.cloud/develop_an_application/mendix/user_journey.html).
```
Comment (SharathNShyam) [inline content/en/docs/developerportal/deploy/mendix-app-on-industrial-edge.md:22]: In order to guarantee the best experience, you will need to set up your system accordingly.
Check out the following Link to understand dependencies and prerequisites to use Mendix on Edge

The above text is required under Prerequistes section.

File: content/en/docs/developerportal/deploy/mendix-app-on-industrial-edge.md [hunk 2]
```diff
@@ -10,216 +10,34 @@ tags: ["Mendix app", "Industrial Edge", "edge", "docker image", "Mendix Buildpac
...
```
Comment (SharathNShyam) [inline content/en/docs/developerportal/deploy/mendix-app-on-industrial-edge.md:21]: Replace the link with the below one:

(https://docs.eu1.edge.siemens.cloud/develop_an_application/mendix/overview.html)

File: content/en/docs/developerportal/deploy/mendix-app-on-industrial-edge.md [hunk 3]
```diff
@@ -10,216 +10,34 @@ tags: ["Mendix app", "Industrial Edge", "edge", "docker image", "Mendix Buildpac
...
 
-    You need the following versions of the Industrial Edge App Publisher:
-    * IEAP UI: minimum v1.2.8
-    * IEAP CLI: minimum v1.3.3
-* [Docker](https://docs.docker.com/get-docker/) with [tcp Socket enabled](https://docs.docker.com/engine/reference/commandline/dockerd/) for Docker daemon
-* An Industrial Edge Management (IEM) to deploy your app 
+For more information about pushing your applications to the Industrial Edge Hub, see [Get started](https://docs.eu1.edge.siemens.cloud/get_started_and_operate/industrial_edge_hub/setup/ieh_index.html).
 
-If you want to test and deploy your converted Mendix app, you also need an Industrial Edge Device (IED) on which to install the Mendix app.
+For more information about pushing your applications to the Industrial Edge Management system, see [Industrial Edge Management - Overview](https://docs.eu1.edge.siemens.cloud/get_started_and_operate/industrial_edge_management/overview.html).
```
Comment (SharathNShyam) [inline content/en/docs/developerportal/deploy/mendix-app-on-industrial-edge.md]: please update the link with the following:
(https://docs.eu1.edge.siemens.cloud/develop_an_application/mendix/user_journey.html)

**Review by SharathNShyam — CHANGES_REQUESTED** [review]
I have reviewed and added my comments.

File: content/en/docs/developerportal/deploy/mendix-app-on-industrial-edge.md (hunk 2 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/developerportal/deploy/mendix-app-on-industrial-edge.md:21]: The sense of the paragraph is the same, but my version is rewritten to match our standard documentation style. The original text does not match some of our requirements, so I can't use it directly, but I think my rewritten version covers the meaning.

File: content/en/docs/developerportal/deploy/mendix-app-on-industrial-edge.md (hunk 3 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/developerportal/deploy/mendix-app-on-industrial-edge.md]: Done.

**Comment by katarzyna-koltun-mx** [comment]
@SharathNShyam I have made 1 requested change and decided not to make the other one, see my reasoning above. I've also reached out to the Academy contact responsible for the course, and will add the link to the course when it's public. Let me know if I should publish this PR now, or wait until the course is available.

**Pull request merged**